Readme.md

```diff
@@ -47,7 +47,10 @@ IMPORTANT: Every page has to have a `title` and `description` meta field.
 
 The markdown body is parsed as [MDX](https://mdxjs.com/) and can include any valid HTML or JSX.
 
-To inline fragments, and have them conditionally render based off selected platform, we add the condition to the `Fragments` tag:
+### Fragments
+To incorporate new platform-specific content within a page, please use [Inline Filters](https://github.com/aws-amplify/docs/blob/main/Readme.md#inline-filters).
+
+To inline existing fragments, and have them conditionally render based off selected platform, we add the condition to the `Fragments` tag:
```

Comment:
Can you please incorporate something like: "When editing content that hasn't been migrated, you may see the following pattern: "